index.js

```diff
@@ -25,7 +25,7 @@ const main = async () => {
   }
 
   await browser.close();
-  console.table(statuses);
+  console.log(statuses);
```

Comment:
Why this change? `console.log` will log the raw object which I don't think is a good UX.

Reply:
If this is to support Node versions prior to v10.0.0, we can use something like [this](https://stackoverflow.com/a/20798760/4176188) and use `console.log` for unsupported versions.